app/assets/javascripts/components/InvitationForm.coffee

```diff
@@ -11,12 +11,17 @@ invitationForm = React.createClass
 
   submit: (enteredValues) ->
     { dispatch } = Screensmart.store
-
-    # Place domainId value in an array. To be removed in the future when multiple domains
-    # can be chosen.
-    enteredValues.domainIds = [enteredValues.domainIds]
     dispatch Screensmart.Actions.sendInvitation(enteredValues) if @props.valid
 
+  toggleDomain: (domainId, checked) ->
+    { fields: { domainIds } } = @props
+    if checked
+      domainIds.addField(domainId)
```

Comment:
Normaal gesproken een goed idee (stuk simpeler), maar Redux werkt met immutable datastructuren waardoor je `domainIds` niet zomaar aan mag passen. Dit betekent dat elke wijziging aan je datastore doet d.m.v een action (`actions.coffee`) en reducer (`reducers/index.coffee`). redux-form doet het in dit geval voor ons met een ingebouwde actie als je `addField` gebruikt.